### DISCUSSION TITLE
How to proxy static files?

### DISCUSSION THREAD

**Discussion by jackshang** [apache/apisix#5789] [Q&A]
咨询下 Apisix 怎么代理静态文件的，是要自己写插件，还是现在通过配置就可以了？ 
如果自己写，通过匹配特定的URL和文件后缀，然后读取指定文件夹里的静态文件，不知道是否是合理的实现？

**Comment by juzhiyuan** [comment]
Hi @jackshang, here has a similar issue, please take a look first: https://github.com/apache/apisix/issues/5792

**Reply by jackshang** [reply]
After reading the thread by you, I'm as confuse as them.
Modifying the configuration file should be just a workround with the Apisix, especially if there are multiple sites(domain).
Although this requirement is not very important, it's very common.
